### ISSUE TITLE
Explain the features in readme

### ISSUE THREAD

**Issue by vranki** [matrix-hacks/matrix-puppet-facebook#46]
This looks like interesting project, but the documentation only explains how to install and set it up.

It would be useful to also tell what it can do and how to use it. Can it bridge fb messages? Can it bridge fb groups, pages or events? How to actually use it after installing from Matrix side?

These should go to readme or separate user's guide linked from readme.

**Comment by thomas-profitt** [comment]
Agreed that that the capabilities of the bridge, and of the other bridges, should be made more clear.

In the meantime, let me try and answer your questions here, as a matrix-puppet-bridge user not familiar with facebook or matrix-puppet-facebook:
- The bridge lets you use a personal matrix homeserver, the bridge itself, and your matrix client as a Facebook chat ("messenger?" is that something different?) client; what your matrix user says is said by your facebook user, and what other facebook users say is said by "ghost" users in Matrix that represent them.
- Only chat is bridged, not groups, pages or events.
- To use the bridge, just wait until you receive a message from another facebook user, on facebook. The bridge, if it's properly configured, will create a room and a ghost user and that user acts as the other user, so you'll talk to them as if they were matrix users and they talk to your facebook user.
- You can't initiate direct-messages, at least not yet.

If I'm wrong about any of that and a matrix-puppet-facebook user has a correction, I'm all ears.

**Comment by vranki** [comment]
Thanks, this would be a good clarification on the capabilities. 

As a potential user it would be interesting to know what is the roadmap. AFAIK chat has some kind of API. Do groups, pages or events have such API and is there any plan to support those? I suppose it's not plausible unless they have APIs. This could be also included in readme.
